server.deno.js

```diff
@@ -4,8 +4,19 @@ Deno.serve(async (req) => {
   const pathname = new URL(req.url).pathname;
   console.log(pathname);
 
+  const req_qiita = await new Recuest(
```

Comment:
```suggestion
  const reqQiita = await new Recuest(
```
JacaScriptなのでlowerCamelCaseで書くとよさそう！
https://qiita.com/deerboy/items/f035b9044edf9a51aff7